doc/user/content/sql/types/list.md

```diff
@@ -237,10 +237,10 @@ Layered slices
 {{< /experimental >}}
 
 To slice on layers beyond the first, indicate the range you want to slice along
-each layer, separating the ranges with commas:
+each layer, separating the ranges with square brackets:
```

Comment:
Is this a backward incompat change that needs to be called out in release notes?

Reply:
Experimental changes don't need to get called out in release notes bc they're allowed to be undocumented breaking behavior.